build/Helix/ConvertWttLogToXUnit.cs

```diff
@@ -288,5 +333,23 @@ public static TestPass ParseTestWttFile(string fileName, bool cleanupFailuresAre
                 return testpass;
             }
         }
+
+        private static string AttributeValue(XElement element, string attributeName)
```

Comment:
>AttributeValue [](start = 30, length = 14)

Will GetAttributeValue be better than this?

Reply:
This handles the case where there is no matching attribute to avoid a null deref error.

Reply:
Agree with your proposed name change. Will update.